CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ if(POLICY CMP0116)
   cmake_policy(SET CMP0116 OLD)
 endif()
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON CACHE BOOL "")
```

Comment:
We might want to explicitly set `LLVM_ENABLE_PIC` on just to be safe?

Reply:
I added this because MIOpen on Windows complained some symbols could not be relocated. The symbol was from rocMLIR, not from LLVM. If you insist, I will add the flag `LLVM_ENABLE_PIC` to `cmake\llvm-project.cmake`.

Reply:
It's default on, so yeah, this shouldn't be a big deal.